### PR TITLE
Remove old code

### LINKED ISSUE #1652: Remove python stats code
Once rust stats code lands, python stats infrastructure should be removed.

### PR DESCRIPTION
**Pull request by jgrund**
Partly Fixes #1652.

This removes code from manager end. Still need to remove code from agent end.

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1691)

### CODE REVIEW

File: tests/unit/chroma_api/test_query_scaling.py [hunk 1]
```diff
@@ -248,5 +248,4 @@ def test_filesystem_targets(self):
         self.assertEqual(target_scaling.queries_per_object, QUERIES_PER_TARGET)
 
         filesystem_scaling = self._measure_scaling(self._create_filesystem_n_osts, FilesystemResource, TargetResource)
-        self.assertIsInstance(filesystem_scaling, OrderN)
-        self.assertEqual(filesystem_scaling.queries_per_object, QUERIES_PER_FILESYSTEM_TARGET)
+        self.assertIsInstance(filesystem_scaling, Order1)
```
Comment (johnsonw) [inline tests/unit/chroma_api/test_query_scaling.py]: Is this metric related?

Comment (jgrund) [reply tests/unit/chroma_api/test_query_scaling.py]: yeah, apparently removing the metric queries improved the runtime.

Comment (johnsonw) [reply tests/unit/chroma_api/test_query_scaling.py]: cool 👍
